.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,19 @@
+name: "Pull Request Labeler"
```

Comment:
Btw there is a bot that I wrote for this and other boooring activities: https://github.com/kaxil/boring-cyborg which could be used as well. We use that in Airflwo and tensorflow

Reply:
okay nice, will try adding this.

Reply:
I tried installing the app. But looks like it sent a request for approval to be added to Astronomer. Maybe we need to wait until it gets approved.